fhir-search/src/main/java/com/ibm/fhir/search/location/uom/standard/StatuteLengthUnits.java

```diff
@@ -16,6 +16,10 @@
 public enum StatuteLengthUnits {
     FOOT("ft_us", 1.0),
     FOOT_CI("FT_US", 1.0),
+    // Support for common variants that are not in UCUM
+    FOOT_VARIANT_FTS("fts", 1.0),
+    FOOT_VARIANT_FOOT("foot", 1.0),
+    FOOT_VARIANT_FEET("feet", 1.0),
```

Comment:
not `ft`?

Reply:
thanks, it's hard to see the tree in the forest